.github/workflows/test_stubgenc.yml

```diff
@@ -0,0 +1,31 @@
+name: Test stubgenc on pybind11-mypy-demo
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+  pull_request:
+    paths:
+    - 'mypy/stubgenc.py'
+    - 'mypy/stubdoc.py'
+    - 'test-data/stubgen/**'
```

Comment:
This should also run for changes in `misc/test-stubgenc.sh`.

Reply:
I'll address this in #9877